lib/dynflow/persistence_adapters/sequel_migrations/023_sqlite_workarounds.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+Sequel.migration do
+  up do
+    if database_type == :sqlite && SQLite3::SQLITE_VERSION == "3.7.17"
```

Comment:
should we have that specific version? wouldn't it be better to have older than?
Or just drop it on SQLite always? There is not much value in having constraints that works part of the time I guess.